rabbus.go

```diff
@@ -219,24 +220,27 @@ func (r *rabbus) register() {
 }
 
 func (r *rabbus) produce(m Message) {
-	body, err := json.Marshal(m.Payload)
-	if err != nil {
+	if err := r.ch.ExchangeDeclare(m.Exchange, m.Kind, r.config.Durable, false, false, false, nil); err != nil {
 		r.emitErr <- err
 		return
 	}
 
+	if m.ContentType == "" {
+		m.ContentType = "application/json"
```

Comment:
I would use constants for the common types.

Something like:
Json = 'application/json'
Plain = 'text/plain'